apps/libsignpost/signpost_energy_monitors.c

```diff
@@ -0,0 +1,186 @@
+#include "i2c_selector.h"
+#include "ltc294x.h"
+#include "signpost_energy_monitors.h"
+#include "max17205.h"
+
+static uint8_t module_num_to_selector_mask[8] = {0x4, 0x8, 0x10, 0, 0, 0x20, 0x40, 0x80};
+static ltc294x_model_e model;
+static uint16_t prescaler;
+
+
+////////////////////////////////////////////////////////////
+//Initialization functions
+//Make sure to call the right one
+/////////////////////////////////////////////////////////
+void signpost_energy_init_ltc2941 (void) {
+    // configure each ltc with the correct prescaler
+    for (int i = 0; i < 8; i++) {
+        i2c_selector_select_channels_sync(1<<i);
+        ltc294x_configure_sync(LTC2941, InterruptPinAlertMode, POWER_MODULE_PRESCALER_LTC2941, VbatAlertOff);
+    }
+
+    model = LTC2941;
+    prescaler = POWER_MODULE_PRESCALER_LTC2941;
+
+    // set all channels open for Alert Response
+    i2c_selector_select_channels_sync(0xFF);
+}
+
+void signpost_energy_init_ltc2943 (void) {
+    // configure each ltc with the correct prescaler
+    for (int i = 0; i < 9; i++) {
+        i2c_selector_select_channels_sync(1<<i);
+        ltc294x_configure_sync(LTC2943, InterruptPinAlertMode, POWER_MODULE_PRESCALER_LTC2943, ADCScan);
+    }
+
+    model = LTC2943;
+    prescaler = POWER_MODULE_PRESCALER_LTC2943;
+
+    // set all channels open for Alert Response
+    i2c_selector_select_channels_sync(0x1FF);
+
+    //max17205_configure_pack_sync();
```

Comment:
What are you questioning? The configure pack? I should remove that - it no longer exists.